src/layout/organize/JourneyInstanceLayout.tsx

```diff
@@ -1,83 +1,137 @@
-        <Box
-          style={{
-            alignItems: 'center',
-            display: 'flex',
-          }}
-        >
-          <JourneyStatusChip instance={journeyInstance} />
-          <Typography style={{ marginRight: '1rem' }}>
-            <Msg id="layout.organize.journeys.lastActivity" />{' '}
-            <ZetkinRelativeTime datetime={journeyInstance.updated} />
-          </Typography>
-          {journeyInstance.next_milestone && (
-            <>
-              <ScheduleIcon
-                color="secondary"
-                style={{ marginRight: '0.25rem' }}
-              />
-              <Typography>
-                {journeyInstance.next_milestone.title}
-                {': '}
-                {journeyInstance.next_milestone.deadline && (
-                  <FormattedDate
-                    day="numeric"
-                    month="long"
-                    value={journeyInstance.next_milestone.deadline}
-                    year="numeric"
-                  />
-                )}
-              </Typography>
-            </>
-          )}
-        </Box>
-      }
-      tabs={[
-        {
-          href: '/',
-          messageId: 'layout.organize.journeys.tabs.timeline',
-        },
-        {
-          href: '/milestones',
-          messageId: 'layout.organize.journeys.tabs.milestones',
+  const journeysQuery = journeysResource(orgId as string).useQuery();
+  const journeys = journeysQuery.data;
+
+  const journeyInstanceHooks = journeyInstanceResource(
+    orgId as string,
+    instanceId as string
+  );
+  const patchJourneyInstanceMutation = journeyInstanceHooks.useUpdate();
+
+  const ellipsisMenu: ZetkinEllipsisMenuProps['items'] = [];
+
+  const submenuItems =
+    journeys
+      ?.filter((journey) => journey.id.toString() !== journeyId)
+      .map((journey) => ({
+        id: `convertTo-${journey.id}`,
+        label: journey.singular_label,
+        onSelect: () => {
+          patchJourneyInstanceMutation.mutateAsync(
+            {
+              journey_id: journey.id,
+            },
+            {
+              onError: () => showSnackbar('error'),
+              onSuccess: () =>
+                router.push(
+                  `/organize/${orgId}/journeys/${journey.id}/${instanceId}`
+                ),
+            }
+          );
         },
-      ]}
-      title={
-        <>
-          {`${journeyInstance.title || journeyInstance.journey.title} `}
-          <Typography
-            color="secondary"
-            variant="h3"
-          >{`\u00A0#${journeyInstance.id}`}</Typography>
-        </>
-      }
-    >
-      {children}
-    </TabbedLayout>
+      })) ?? [];
+
+  if (submenuItems.length) {
+    ellipsisMenu.push({
+      id: 'convert-journey',
+      label: intl.formatMessage({
+        id: 'pages.organizeJourneyInstance.ellipsisMenu.convert',
+      }),
+      startIcon: <Forward color="secondary" />,
+      subMenuItems: submenuItems,
+    });
+  }
+
+  return (
+    <>
```

Comment:
This fragment seems redundant to me (and has caused a big redundant diff because it indents everything unnecessarily).